providers/redhat/schema/convertutils.go

```diff
@@ -68,7 +70,7 @@ func IsFixed(fixState string) bool {
 		return true
 	default:
 		log.Printf("unknown fix state: %q", fixState)
-		return true
+		return false
```

Comment:
Both strategies seems reasonable enough, reducing noise vs. increasing awareness... both can be useful. I'd make it a argument of the function, something like `IsFixed(fixState string, default bool) bool {...}`

Reply:
There are 2 users of this function and I can't see us wanting to return false in any of the call sites. Considering the domain (security) a false position seems always better than a false negative where we can miss security problems.

On a different note, I think there are two modes of operation that are interesting but on a slightly different axis:

- A strict mode where we error out if we don't recognise the string
- A loose mode where we make the decision to warn and return it's isn't fixed

But we don't really need the strict mode at the moment. It could be useful to run the strict mode from time to time to ensure we fully parse the data from the feed (data that can evolve over time as well).

Reply:
fair enough, let's merge it.